draft-trammell-rains-protocol.md

```diff
@@ -1682,8 +1656,9 @@ To generate a canonicalized P-Shard:
 
 To generate a canonicalized Message:
 
-- preserve the order of Sections within the Message
-- canonicalize each section as appropriate by following the canonicalization
+- preserve the order of the Sections (Assertions, Queries and Notifications)
```

Comment:
What is the value of this change? The intended meaning is that the order of Sections within a Message is significant; adding types of sections sounds, to me, like the intention is to order sections by type?

Reply:
We never state what a 'Section' is. Better define it properly when we introduce Messages and remove my additions here